flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/functions/AdvancedFunctionsExample.java

```diff
@@ -55,7 +56,11 @@ public static void main(String[] args) throws Exception {
         final EnvironmentSettings settings =
                 EnvironmentSettings.newInstance().inBatchMode().build();
         final TableEnvironment env = TableEnvironment.create(settings);
-
+        env.getConfig()
```

Comment:
why this?

Reply:
`because: AdvancedFunctionsExampleITCase will print out with `,`  instead of `, `
since the default config is LEGACY enabled.

I kind of preferred to show the row printing with the new  format.
I will add a comment to explain why we need this.

Reply:
As discussed, reverted this, so we continue use the default behaviour (currently) which is the legacy behaviour, therefore `,` instead of `, `. Adjusted the printout in the assertions accordingly.